drgn/internal/cli.py

```diff
@@ -4,18 +4,15 @@
 """drgn command line interface"""
 
 import argparse
-import builtins
-import code
-import importlib
 import os
 import os.path
 import pkgutil
 import runpy
-import shutil
 import sys
 from typing import Any, Dict
 
 import drgn
+from drgn.cli import drgn_version_header, run_cli
```

Comment:
Rather than splitting this into two modules, I'd prefer to move it all to `drgn.cli` and make everything other than `run_interactive()` "private" (i.e., prefix it with "_").